virt/support/virt-troubleshooting.adoc

```diff
@@ -6,6 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+You can troubleshoot {VirtProductName} components by using the tools provided in the web console or by using the `oc` CLI tool.
```

Comment:
I recommend you add a link to "tools provided in the web console" pointing to the web console section of the Support overview page. Then you can remove the note.